src/sys/IFACE.ASM

```diff
@@ -11,8 +11,8 @@
 ; ========================================================================
 
 ; GET KEYBOARD INPUT: Get user input from the keyboard, and store in the buffer provided
-; IN: CX = Buffer size, ES:DI = Location of buffer
-; OUT: DI = Initial value + number of characters input
+; IN: CX = Buffer size; ES:DI = Location of buffer
+; OUT: DI = Initial value + number of characters input; CF = Set if CTRL+BREAK
```

Comment:
CTRL+BREAK was incorporated in order to allow for empty lines when typing teletype input for a program.